### PR TITLE
Circleci

### PR DESCRIPTION
**Pull request by ifrit98**
* Add black format check
* Add compatibility check for python deps
* Add build check

### CODE REVIEW

**Review by Eugene-hu — COMMENTED** [review]
Is this update just for adding circleci and running black?

File: requirements.txt
Comment (steffencruz) [inline requirements.txt:1]: good catch

File: neurons/miner.py
Comment (steffencruz) [inline neurons/miner.py:1]: looks like the diff is just black?

**Review by steffencruz — APPROVED** [review]
Looks good!

**Pull request merged**